src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTests.cs

```diff
@@ -747,8 +747,8 @@ public void Roundtrip_CrossProcess(object obj)
             }
         }
 
-        [ActiveIssue(16753)] //Fails on desktop and core: 'Unable to cast object of type 'System.UInt32[][*]' to type 'System.Object[]'
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
> I can just delete the test if that is preferred, or make it a negative test.

We shouldn't delete the test.  If we're uncomfortable fixing a bug like this, then the answer isn't to delete the test, but rather to not fix the bug.  I'm not sure why we wouldn't fix it, though.